index.bs

```diff
@@ -1610,7 +1914,253 @@ level APIs. The list above includes interface classes for which such high level
 APIs exist and provide greater protection for user privacy and security than
 low level access through this API would.
 
-## Endpoints ## {#endpoints}
+### Constructors ### {#usbinterface-constructors}
+
+<dl dfn-type=constructor dfn-for="USBInterface">
+    : <dfn>USBInterface(configuration,interfaceNumber)</dfn>
+    ::
+        <div algorithm="USBInterface constructor"> 
+            1. Let |configuration| be the parameter of <code>configuration</code>.
+            1. Let |interfaceNumber| be the parameter of <code>interfaceNumber</code>.
+            1. Set {{USBInterface/[[configuration]]}} to |configuration|.
+            1. Set {{USBInterface/[[interfaceNumber]]}} to |interfaceNumber|.
```

Comment:
Always specify what object your are setting an internal slot on.

```suggestion
            1. Set <a>this</a>.{{USBInterface/[[configuration]]}} to |configuration|.
            1. Set <a>this</a>.{{USBInterface/[[interfaceNumber]]}} to |interfaceNumber|.
```

Reply:
Updated in https://github.com/WICG/webusb/pull/212/commits/1a96f493fc285da99d2b812d809c241388197beb.